### PR TITLE
Prometheus: Update default query type option to "Both" in Explore query editor

### PR DESCRIPTION
**Pull request by ivanahuckova**
**What this PR does / why we need it**:
This PR updates default option for Prometheus query type in Explore to **Both**.  It also adds tooltip with explanation what query type means.

### CODE REVIEW

File: public/app/plugins/datasource/prometheus/components/PromExploreExtraField.tsx [hunk 1]
```diff
@@ -34,7 +34,12 @@ export const PromExploreExtraField: React.FC<PromExploreExtraFieldProps> = memo(
           )}
           aria-label="Query type field"
         >
-          <InlineFormLabel width={5}>Query type</InlineFormLabel>
+          <InlineFormLabel
+            width="auto"
+            tooltip="Choose the type of query you would like to run. With instant queries, you query only the latest value. Range query is used to do a query over a range of time. With both, you run 2 queries - instant and range."
```
Comment (ivanahuckova) [inline public/app/plugins/datasource/prometheus/components/PromExploreExtraField.tsx:39]: @oddlittlebird could you please look at the content of this tooltip.

**Comment by aocenas** [comment]
@ivanahuckova in context of the discussion around loki and adding the same switch while removing the `both` options, is there anything we want to align between loki/prom? I mean no feelings here maybe it does not make sense to align them just seemed non intuitive that we won't have `both` in loki while it will be default here.

**Comment by ivanahuckova** [comment]
Do you see any other reasons to do that (besides making Loki & Prometheus aligned in this)? Because I think this is useful option and there is no problem with having it (compared to Loki where `both` option didn't make sense for logs queries). So my opinion would be keeping all of these.

@gouthamve @beorn7 do you have any preferences regarding keeping 3 query type options in Explore `instant-range-both` versus just 2 - `instant-range` ?

**Comment by beorn7** [comment]
Not feeling strongly either way, but from a pure Prometheus perspective, I think the _Both_ default is a good starting point. With _Range_ as the default, I'd assume many users are confused about the _Table_ section being humongous with all the different timestamps. With _Instant_ as the default, it's disappointing to not see any graph.

File: public/app/plugins/datasource/prometheus/components/PromExploreExtraField.tsx (hunk 1 above)
Comment (oddlittlebird) [reply public/app/plugins/datasource/prometheus/components/PromExploreExtraField.tsx:39]:
```suggestion
            tooltip="Choose the type of query you would like to run. An instant query queries only the latest value. A range query queries over a range of time. With both, you run two queries - one instant and one range."
```

File: public/app/plugins/datasource/prometheus/components/PromExploreQueryEditor.tsx [hunk 2]
```diff
@@ -57,7 +57,8 @@ export const PromExploreQueryEditor: FC<Props> = (props: Props) => {
       data={data}
       ExtraFieldElement={
         <PromExploreExtraField
-          queryType={query.range && query.instant ? 'both' : query.instant ? 'instant' : 'range'}
+          // Select "both" as default option when Explore is opened
+          queryType={query.range === query.instant ? 'both' : query.instant ? 'instant' : 'range'}
```
Comment (aocenas) [inline public/app/plugins/datasource/prometheus/components/PromExploreQueryEditor.tsx:61]: This `query.range === query.instant` is weird and not very intuitive. I assume false === false cannot happen, but still I think `query.range && query.instant` makes more sense when reading it.

Comment (ivanahuckova) [reply public/app/plugins/datasource/prometheus/components/PromExploreQueryEditor.tsx:61]: Added comments to explain this

**Review by zoltanbedi — APPROVED** [review]
lgtm 🍍

**Review by Elfo404 — APPROVED** [review]
Not sure about the wording in https://github.com/grafana/grafana/pull/28935/files#diff-37b961f1cd1fc28385e28aa6de84bb2e4efffc82acfe0333fc5b5ad8e2d63bc7R39

but looks good to me!

**Pull request merged**